recipes/easyloggingpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["crash_log_enabled"] = self.options["easyloggingpp"].enable_crash_log
```

Comment:
Why not export it on `cpp_info.defines` ?

Reply:
thanks @uilianries @madebr this approach is even cleaner!